src/views/firearm.njk

```diff
@@ -0,0 +1,154 @@
+{% extends "_base.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% set dateErrors %}
+  {% if model.certificateIssuedDayError %}
+    <p>Enter a valid day</p>
+  {% elif model.certificateIssuedMonthError %}
+    <p>Enter a valid month</p>
+  {% elif model.certificateIssuedYearError %}
+    <p>Enter a valid year</p>
+  {% elif model.certificateIssuedFutureError %}
+    <p>The date your firearms certificate was issued must be in the past</p>
+  {% elif model.certificateIssuedExpiredError %}
+    <p>The date your firearms certificate was issued must be less than five years old</p>
+  {% else %}
+    <p>Enter a valid date</p>
+  {% endif %}
+{% endset -%}
+
+{% block content %}
+
+  {% if model.firearmError %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+            text: "You must enter the number found on your firearm certificate.",
+            href: "#certificate-number-error"
+        } if model.certificateNumberError,
+        {
+            text: "You must enter the date you aquired your firearm certificate.",
+            href: "#certificate-issued-error"
+        } if model.certificateIssuedInvalidError,
+        {
+            text: "Please enter a valid year.",
+            href: "#certificate-issued-error"
+        } if model.certificateIssuedYearError,
+        {
+            text: "Please enter a valid month.",
+            href: "#certificate-issued-error"
+        } if model.certificateIssuedMonthError,
+        {
+            text: "Please enter a valid day.",
+            href: "#certificate-issued-error"
+        } if model.certificateIssuedDayError,
+        {
+            text: "Your firearm certificate issue date can not be a future date.",
+            href: "#certificate-issued-error"
+        } if model.certificateIssuedFutureError,
+        {
+            text: "Your firearm certificate must be less than five years old.",
+            href: "#certificate-issued-error"
+        } if model.certificateIssuedExpiredError
```

Comment:
Following their lead...

https://design-system.service.gov.uk/components/date-input/#if-the-date-is-in-the-future-when-it-needs-to-be-in-the-past

> **If the date is in the future when it needs to be in the past**
> Highlight the date input as a whole.
> Say ‘[whatever it is] must be in the past’. 

https://design-system.service.gov.uk/components/date-input/#if-the-date-must-be-after-another-date

> **If the date must be after another date**
> Highlight the date input as a whole.
> Say ‘[whatever it is] must be after [date and optional description]’.

...I think
'The date your firearm certificate was issued must be in the past'
and
'The date your firearm certificate was issued must be after 16 September 2015'

are probably closer to what GDS want from us.

Reply:
Resolved in b86955a.